hibernate-core/src/main/java/org/hibernate/boot/model/internal/PropertyBinder.java

```diff
@@ -1466,4 +1471,20 @@ private static void removeIdGenerators(List<Annotation> generatorAnnotations, Li
 			}
 		}
 	}
+
+	private static BeanContainer getBeanContainer(ServiceRegistry serviceRegistry) {
+		if (!Helper.allowExtensionsInCdi( serviceRegistry )) {
+			ID_GEN_FAC_LOGGER.debug( "Ignoring CDI for resolving IdentifierGenerator instances as extended or delayed CDI support was enabled" );
```

Comment:
We should use a different logger here, since this code is not using `IdentifierGeneratorFactory` (and that class has been deleted in H7).

Reply:
I don't think the debug message is useful here, I removed it.